### ISSUE TITLE
DisallowMultiPropertyDefinition creates invalid code for array properties with comments

### ISSUE THREAD

**Issue by kkmuffme** [slevomat/coding-standard#1433]
v8.4.0

```xml
<ruleset name="testing">
    <rule ref="SlevomatCodingStandard.Classes.DisallowMultiPropertyDefinition" />
</ruleset>
```

for this code:

```php
class Hello {
	/**
	 * foo
	 *
	 * @var string[]
	 */
	public $world = array(
		'details',
		'preload', // some details
	);
}
```

phpcbf creates invalid PHP code:

```php
class Hello {
	/**
	 * foo
	 *
	 * @var string[]
	 */
	public $world = array(
		'details';
	/**
	 * foo
	 *
	 * @var string[]
	 */
	public 'preload';
	/**
	 * foo
	 *
	 * @var string[]
	 */
	public );
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/2695d1c4a7cb68f02c0142cf7bc7912e90b39d6e
